### ISSUE TITLE
Argument "max.attributes" is missing, with no default

### ISSUE THREAD

**Issue by pcarbo** [hhoeflin/hdf5r#91]
I'm enjoying the new OOP hdf5 interface in R. However, following the simple example provided in the README, I encounter an error when trying to display an hdf5 data object in R.

```R
library(hdf5r)
data(cars)
f <- H5File$new("cars.h5",mode = "w")
f[["cars"]] <- cars
x           <- f[["cars"]]
h5attr(x,"rownames") <- rownames(cars)
print(x)
# Error in print_attributes(self, max_to_print = max.attributes) :
# argument "max.attributes" is missing, with no default
```

I think it may be a matter of adding a default value for the `max.attributes` argument to the `print` method. The standard print interface in R should accept printing of the object `x` without any additional arguments. Is that possible? I suppose this is a relatively minor issue, but I posted it only because `print` is used often. Thank you.

**Comment by hhoeflin** [comment]
This looks like a duplicate of #89 and should be fixed in the master branch. Could you install the newest version and check that it works for you?

**Comment by pcarbo** [comment]
@hhoeflin Apologies for not noticing this earlier issue. I was able to install the master branch successfully, and the error now goes away.

On a related note, when installing from source, I noticed that your installation instructions could use a bit more detail. In particular, many users will have to modify the R compiling and/or linking settings when the hdf5 library is installed in a non-standard location. For example, I have installed the hdf5 library using Homebrew on my MacBook Pro, and I added `LDFLAGS = -L/usr/local/Cellar/hdf5/1.10.1_2/lib` to my `~/.R/Makevars` file before installing `hdf5r` using `devtools`.
